proposals/nnnn-ios-update-minimum-version-ios10.md

```diff
@@ -0,0 +1,35 @@
+# Update the Minimum Required iOS Version to 10.0
+* Proposal: [SDL-NNNN](nnnn-ios-update-minimum-version-ios10.md)
+* Author: [Joel Fischer](https://github.com/joeljfischer)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Java Suite / HMI / Policy Server / SHAID / RPC / Protocol]
```

Comment:
Is this accurate?  I only see iOS described throughout the proposal.

Reply:
No, ha, I somehow missed that line